tests-web/assets/js/tests.js

```diff
@@ -49,7 +49,8 @@ export const loadTests = async () => {
     for (const t of testNames) {
         const li = listNode.appendChild(window.document.createElement('li'));
         const a = li.appendChild(window.document.createElement('a'));
-        a.href = '/t/' + t;
+        // Here we encode the test name in the URL to be read elsewhere.
+        a.href = '?test=' + t;
```

Comment:
It is often nice to implement the encoding/decoding of app state to/from a URL in the same place to hide the details (which makes it easy to retain additional params and that sort of thing). But for now, this is fine.